plugins/shared/cmd/launcher/command/device.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/go-multierror"
```

Comment:
This line should appear in following grouping of imports, along with other non-stdlib imports.  I'd suggest configuring your IDE to use `goimports` on save instead of plain `gofmt`.